### ISSUE TITLE
How do I get a client_id and client_secret? Is there a dummy wallet I can use for test / dev?

### ISSUE THREAD

**Issue by dickhardt** [stateofca/opencred#2]
Title says it all. @msporny

**Comment by brianorwhatever** [comment]
The client_id and client_secret for a relying party should be generated independent of OpenCred and supplied to OpenCred via the config. At this point there is not any automatic generation in place.

Any wallet that speaks OID4VP or VC-API should be able to present credentials to OpenCred -- I don't know of any dummy wallets available at this time

**Comment by dickhardt** [comment]
I got here from this page https://www.dmv.ca.gov/portal/ca-dmv-wallet/opencred-for-relying-parties/

And want to access the CA DMV wallet

**Comment by deshmukhrajvardhan** [comment]
I got here from the same site.

Is there an example where we can see the components of this repo in work? Some demo video of all parts working, so that we can aim to reproduce it?

When you say any wallet? From experience entities that create their own Issuer, wallet and verifier interop with only their implementations and not others.
Here is the one I have used in the past (Sphereon has all 3) https://github.com/Sphereon-Opensource/mobile-wallet

**Comment by deshmukhrajvardhan** [comment]
If I understand this repo right, it doesn't call out to the CA DMV database to check if the DL creds are valid. Is this assumption right?

**Comment by brianorwhatever** [comment]
@deshmukhrajvardhan Yes you are correct, OpenCred does not communicate with any DMV database. OpenCred can be deployed to verify DL creds that are presented from the CA DMV Wallet App.

Demo videos are a work in progress so at this point the readme in this repository is the best source of information regarding how to deploy OpenCred.

**Comment by deshmukhrajvardhan** [comment]
Thanks @brianorwhatever oh so OpenCred can verify the CA DMV creds (does it do this by validating the signature? where does OpenCred get CA DMV's pub key/did from?).

Thanks! look forward to the demo and will continue to go through the README. Thanks again!

**Comment by deshmukhrajvardhan** [comment]
@dickhardt I was able to get the verifier to work with CA DMV app https://github.com/stateofca/opencred/issues/5 https://github.com/stateofca/opencred/issues/6
